de_novo/conf_template_de_novo.yml

```diff
@@ -21,6 +21,8 @@ merge_max_mismatch_ratio: 0.2
 assembler: spades
 min_length: 500
 busco_set:
+db: "https://genome-idx.s3.amazonaws.com/kraken/k2_standard_16gb_20230605.tar.gz"
```

Comment:
Please add a title to the section - `# contamination filtration`.
Also, can we make the parameter names more informative? e.g. `contamination_db`, `contamination_confidence`